src/main/java/de/tum/in/www1/artemis/domain/participation/Participation.java

```diff
@@ -85,6 +85,22 @@
     @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
     private Set<Submission> submissions = new HashSet<>();
 
+    /**
+     * This property stores the total number of submissions in this participation. Not stored in the database, computed dynamically and used in showing statistics to the user in
+     * the exercise view.
+     */
+    @Transient
+    @JsonProperty
+    private Integer submissionCount;
```

Comment:
This can also be a primitive type. The count should not be a null value, right?

Reply:
It will always be an actual int value. I switched it.